### ISSUE TITLE
Accessibility check for quick forecast component and headers

### ISSUE THREAD

**Issue by eric-gade** [weather-gov/weather.gov#1937]
### Description
Make some accessibility considerations for the quick forecast nav. This might include removing it from screenreaders entirely, or only presenting the datetime as the link

### Acceptance criteria

- [ ] Tests have been implemented or modified to capture the bug
- [ ] Changes have been tested for accessibility

**Comment by colinmurphy01** [comment]
Can you clarify? Are you looking to get this manually reviewed or when you say "check" do you have something in mind?

**Comment by colinmurphy01** [comment]
@eric-gade to assign to someone for manual accessibility review? Post in slack when it's in staging.

**Comment by eric-gade** [comment]
Closed by work done in #2095
